Makefile

```diff
@@ -71,6 +70,32 @@ test-lastfailed:
 .PHONY: test-clean
 test-clean:
 	if [ -f test_db.sqlite3 ]; then rm test_db.sqlite3; fi
+	$(VIRTUAL_ENV)/bin/py.test
+
+.PHONY: coverage
+coverage:
+	$(VIRTUAL_ENV)/bin/py.test --reuse-db --cov --cov-report=html
+
+.PHONY: lint
+lint:
+	. $(VIRTUAL_ENV)/bin/activate && $(NODE_BIN)/polylint
+
+.PHONY: lint-quick
+lint-quick:
+	. $(VIRTUAL_ENV)/bin/activate && $(NODE_BIN)/polylint -SF
+
+.PHONY: po
+po:
+	$(VIRTUAL_ENV)/bin/python manage.py makemessages -d django
+	$(VIRTUAL_ENV)/bin/python manage.py makemessages -d djangojs
+	sed -i 's%#: .*/adhocracy4%#: adhocracy4%' locale/*/LC_MESSAGES/django*.po
+	sed -i 's%#: .*/a4-meinberlin%#: a4-meinberlin%' locale/*/LC_MESSAGES/django*.po
+	msgen locale/en_GB/LC_MESSAGES/django.po -o locale/en_GB/LC_MESSAGES/django.po
+	msgen locale/en_GB/LC_MESSAGES/djangojs.po -o locale/en_GB/LC_MESSAGES/djangojs.po
+
+.PHONY: mo
+mo:
+	$(VIRTUAL_ENV)/bin/python manage.py compilemessages
 
```

Comment:
Hmm, I don't know:
In a4-meinberlin we use `make makemessages` / `make compilemessages` and in other projects we use other names, now if we change this to `make mo` / `make po` we have it less standardized, so I would leave the name or really commit to mo/po and than at some point we also change it in the other makefiles. And: this can also easily break deployment (i think), so i'll say just leave the name.

Reply:
It is in all the makefiles. There are PRs open. It alignes the translating in all projects, because it was annoying that the makefiles did all have different targets.
Why do you think it breaks the deployment?

Reply:
I think we might call compilemessages on deploy. So we have the po files in the repo, and then on deployment we compile it to mo files. I am quite sure it is in the deploy steps but don't remember where that is.

Reply:
I also couldn't find it. I think it was used in the build-legacy. And now we use the make release-target. But, as the build-legacy should also still work, you're right: I should rename it again and then check opin and ae (where this didn't exist), how it was done there.